README.md

```diff
@@ -13,22 +14,50 @@ docker run --rm \
   -v /tmp/.X11-unix:/tmp/.X11-unix \
   -v /dev/bus/usb:/dev/bus/usb \
   -v /usr/share/X11/xkb:/usr/share/X11/xkb:ro \
-  -v ~/.AndroidStudio:/home/developer/.AndroidStudio \
-  -v ~/.AndroidStudio.android:/home/developer/.android \
   -v ~/.AndroidStudio.java:/home/developer/.java \
-  -v ~/.AndroidStudio.gradle:/home/developer/.gradle \
-  -v ~/.AndroidStudio.sdk:/home/developer/Android/Sdk \
-  -v ~/Project:/home/developer/Project \
-  --name androidstudio-$(head -c 4 /dev/urandom | xxd -p)-$(date +'%Y%m%d-%H%M%S') \
-rycus86/androidstudio:3.4
+  -v ~/.AndroidStudio.config:/home/developer/.config \
+  -v ~/.gradle:/home/developer/.gradle \
+  -v ~/Android/:/home/developer/Android/Sdk \
+  -v "$PWD:/home/developer/Project" \
+  --name androidstudio-$(head -c 4 /dev/urandom | xxd -p)-$(date +'%Y%m%d-%H%M%S')  \
+chubaoraka/docker-androidstudio
```

Comment:
Did you mean to point this at your DockerHub account?

Reply:
Yes, because I uploaded the [Arctic Fox image](https://hub.docker.com/layers/docker-androidstudio/chubaoraka/docker-androidstudio/2020.3.1.26/images/sha256-6d2ea74173d54516e1bbd186cc95367568a8e7d2197ab804841c048f86c8923e?context=explore) on my account.

Reply:
That makes sense. I'd perhaps propose archiving this repo with some pointers to your repo if you'd be interested in maintaining this image? I no longer do Android dev, so it's unlikely I'll bump this often... 

What do you think? If you're happy with that, perhaps replace this PR with one that changes the Readme only to clearly point to your repo, and we could get that in  (and then I could archive this repo)